source/configure/environment-configuration-settings.rst

```diff
@@ -4159,4 +4159,20 @@ This setting isn't available in the System Console and can only be set in ``conf
 
 +----------------------------------------------------------------------------------------------------------------------+
 | This feature's ``config.json`` setting is ``"EnableAPIChannelDeletion": false`` with options ``true`` and ``false``. |
-+----------------------------------------------------------------------------------------------------------------------+
\ No newline at end of file
++----------------------------------------------------------------------------------------------------------------------+
+
+Enable desktop app developer mode
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+From Desktop App v5.10, this setting enables desktop app developer debugging options, and is disabled by default.
+
+This setting isn't available in the System Console and can only be enabled in ``config.json`` by setting the environment variable ``MM_DESKTOP_DEVELOPER_MODE`` to ``true``.
+
+**True**: Unlocks the following options in the Desktop App for the purposes of troubleshooting and debugging. You should only enable this setting if instructed to by a Mattermost developer.
```

Comment:
@devinbinnie - Once this env var is set to `true`, how would a user access these new Developer Mode settings via the Desktop App? Do the access paths differ by platform? (Linux, Mac, Windows)?

Reply:
It's all in the `View > Developer Tools` menu.